Security/README.md

```diff
@@ -0,0 +1,88 @@
+# Security 
+
+This section contains hands-on assignments that will help you secure an AKS workload and is based on the Third Edition of the free book [Hands-on Kubernetes on Azure](https://aka.ms/handson-aks-book). You will start with a simple application deployed following the [Azure Kubernetes Service Workshop](https://docs.microsoft.com/en-us/learn/modules/aks-workshop/) and then proceed to build on it to secure your cluster. You will be working on the following building blocks:
+
+- Network Security in AKS
+- Role Based Access Control on AKS
+- Azure AD pod-managed identities on AKS
+- Storing Secrets in AKS (still under development)
+
+In this tutorial we will be using Azure CLI and Azure Portal to manage the Azure resources. 
+
+## Getting started with the workshop
+
+### Prerequisites
+
+* Enough knowledge in Kubernetes and AKS to understand the [Azure Kubernetes Service Workshop](https://docs.microsoft.com/en-us/learn/modules/aks-workshop/)
+* An Azure Subscription
+* An AKS cluster with the Fruit Smoothie workload already deployed as it is at the end of the AKS workshop 
+* For some of the sections, you will need to have global administrator permissions for the Azure AD instance
```

Comment:
Nobody is going to have Global Admin privileges.  To do this, they would have to stand up their own Azure AD tenant.

Reply:
@rickrain for that secrtion they can use their visual studio subscription, they dont need the fruit smoothie cluster